zebedee-cms/src/main/java/com/github/onsdigital/zebedee/model/Collection.java

```diff
@@ -412,9 +412,8 @@ public static Collection update(
             scheduler.cancel(collection);
         }
 
-        Set<String> teamIds = setViewerTeams(collectionDescription, zebedee, session);
-
         if (collectionDescription.getTeams() != null) {
+            Set<String> teamIds = setViewerTeams(collectionDescription, zebedee, session);
```

Comment:
`teamIds` is not used 🤠  update the setTeams stmt to use or just delete?

Reply:
the side-effect of `setViewerTeams` is the important thing here, but I agree that `teamIds` is not used (but I didn't change that aspect)